exchanges/ftx/ftx_wrapper.go

```diff
@@ -0,0 +1,827 @@
+			tempResp.Amount = triggerOrderData.Result[z].Size
+			tempResp.AssetType = asset.Spot
+			tempResp.Date = triggerOrderData.Result[z].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = triggerOrderData.Result[z].FilledSize
+			tempResp.Pair = currency.NewPairFromString(triggerOrderData.Result[z].Market)
+			tempResp.Price = triggerOrderData.Result[z].AvgFillPrice
+			tempResp.RemainingAmount = triggerOrderData.Result[z].Size - triggerOrderData.Result[z].FilledSize
+			tempResp.TriggerPrice = triggerOrderData.Result[z].TriggerPrice
+			orderVars, err := f.compatibleOrderVars(triggerOrderData.Result[z].Side,
+				triggerOrderData.Result[z].Status,
+				triggerOrderData.Result[z].OrderType,
+				triggerOrderData.Result[z].FilledSize,
+				triggerOrderData.Result[z].Size,
+				triggerOrderData.Result[z].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (f *FTX) GetOrderHistory(getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
+	var resp []order.Detail
+	for x := range getOrdersRequest.Pairs {
+		var tempResp order.Detail
+		orderData, err := f.FetchOrderHistory(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], asset.Spot).String(),
+			getOrdersRequest.StartTicks, getOrdersRequest.EndTicks, "")
+		if err != nil {
+			return resp, err
+		}
+		for y := range orderData.Result {
+			tempResp.ID = strconv.FormatInt(orderData.Result[y].ID, 10)
+			tempResp.Amount = orderData.Result[y].Size
+			tempResp.AssetType = asset.Spot
+			tempResp.ClientOrderID = orderData.Result[y].ClientID
+			tempResp.Date = orderData.Result[y].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = orderData.Result[y].Size - orderData.Result[y].RemainingSize
+			tempResp.Pair = currency.NewPairFromString(orderData.Result[y].Market)
+			tempResp.Price = orderData.Result[y].Price
+			tempResp.RemainingAmount = orderData.Result[y].RemainingSize
+			var orderVars OrderVars
+			orderVars, err = f.compatibleOrderVars(orderData.Result[y].Side,
+				orderData.Result[y].Status,
+				orderData.Result[y].OrderType,
+				orderData.Result[y].FilledSize,
+				orderData.Result[y].Size,
+				orderData.Result[y].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+		triggerOrderData, err := f.GetTriggerOrderHistory(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], asset.Spot).String(),
+			getOrdersRequest.StartTicks, getOrdersRequest.EndTicks, getOrdersRequest.Side.String(), getOrdersRequest.Type.String(), "")
```

Comment:
type and side should be lower as per your test in `TestGetTriggerOrderHistory`, this currently causes an error